src/ImageSharp/Processing/Processors/Overlays/GlowProcessorParallel.cs

```diff
@@ -0,0 +1,98 @@
+﻿// <copyright file="GlowProcessor.cs" company="James Jackson-South">
+// Copyright (c) James Jackson-South and contributors.
+// Licensed under the Apache License, Version 2.0.
+// </copyright>
+
+namespace ImageSharp.Processing.Processors
+{
+    using System;
+    using System.Numerics;
+    using System.Threading.Tasks;
+
+    using ImageSharp.PixelFormats;
+
+    /// <summary>
+    /// An <see cref="IImageProcessor{TPixel}"/> that applies a radial glow effect an <see cref="Image{TPixel}"/>.
+    /// </summary>
+    /// <typeparam name="TPixel">The pixel format.</typeparam>
+    internal class GlowProcessorParallel<TPixel> : ImageProcessor<TPixel>
```

Comment:
I was keeping the old code around while I was rewriting things and forgot to remove. Being removed